app/controllers/registrations_controller.rb

```diff
@@ -4,6 +4,12 @@ def new
     super { |user| user.managers.build.build_nonprofit }
   end
 
+  protected
+  
+    def update_resource(resource, params)
+      resource.update_without_password(params)
+    end
```

Comment:
@ianroberts131 We can discuss the security implication of this change. See feedback at https://github.com/ianroberts131/coalesce-to-give/pull/22/commits/0c62d37e3cccd59f7ae4dbba79c380c39ac38823#r69263101